packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -71,14 +99,44 @@ const Editor = ({
     // publish and unpublish of pipe
     editor?.setOptions({ editable })
   }, [editable])
+  const handleVariableClick = useCallback(
+    (variable: Variable) => {
+      editor?.commands.insertContent({
```

Comment:
is it expected that I need 2 clicks to insert the 1st variable?

Reply:
First thought is that we need to somehow make tiptap give up focus when we mouse over the variables popper